tests/sled_transaction.rs

```diff
@@ -729,3 +732,54 @@ async fn sled_transaction_timeout_index() {
         Ok(select!(id I64; 1))
     );
 }
+
+#[cfg(feature = "metadata")]
+#[test]
+fn sled_transaction_metadata() {
+    macro_rules! test_tables {
+        ($glue: ident []) => {
```

Comment:
Q) [] What is syntax?
I wanted to study, but I haven't found the document yet.
Can you leave a reference?